babel/numbers.py

```diff
@@ -701,8 +719,8 @@ def _format_int(self, value, min, max, locale):
             gsize = self.grouping[1]
         return value + ret
 
-    def _format_frac(self, value, locale):
-        min, max = self.frac_prec
+    def _format_frac(self, value, locale, force_frac=None):
+        min, max = force_frac or self.frac_prec
```

Comment:
@erickwilder that would be an unrelated change and should go into an own commit which is better for maintenance